tests/config/pubsub_components.yaml

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2023 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#kafka pubsub component
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: kafka-messagebus
```

Comment:
Seems like there may be more performance tests for more building blocks and for more components, in future. Should we include `perf` as a prefix/suffix in the naming convention?